docs/reference/master-commandline-reference.md

```diff
@@ -234,6 +232,24 @@ Example:
 nfd-master -extra-label-ns=vendor-1.com,vendor-2.io
 ```
 
+### -deny-label-ns
+
+The `-deny-label-ns` flag specifies a comma-separated list of excluded
+label namespaces. By default, nfd-master allows creating labels in all
+namespaces, excluding `*.kubernetes.io` namespace and its sub-namespaces
+(e.g. `sub.ns.kubernetes.io`). However, you should note that the
+`*.kubernetes.io` label is forcibly denied.
+e.g: `nfd-master -deny-label-ns=""` would still deny `*.kubernetes.io`
+This option can be used to exclude some vendors or application specific namespaces.
+
+Default: *\*.kubernetes.io*
+
+Example:
+
+```bash
+nfd-master -deny-label-ns=kubernetes.io,vendor-2.io
```

Comment:
A bit misleading example as kubernetes.io should be always forcibly denied anyway

```suggestion
nfd-master -deny-label-ns=*.vendor.com,vendor-2.io
```